docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -215,6 +215,56 @@ secrets:
     alluxio-ceph-config: cephConfig
 ```
 
+***Examples: Memory Tiered Store Only***
+
+```properties
+tieredstore:
+  levels:
+  - alias: MEM
+    mediumtype: MEM
+    level: 0
+    path: /dev/shm
+    type: hostPath
```

Comment:
please mention that the 2 supported volume types right now are [hostPath, emptyDir]

Reply:
For `hostPath` do we require root privilege to write to it?

Reply:
According to K8s doc:
> the files or directories created on the underlying hosts are only writable by root. You either need to run your process as root in a privileged Container or modify the file permissions on the host to be able to write to a hostPath volume

So if Alluxio is not run as root, things will be tricky. We either need another `initContainer` running as root to change the permission or explicitly state that the user need to watch out for the permission constraint.

Reply:
yes we do. we will move away from the limitation as part of using local persistent volume in a separate PR

Reply:
@cheyang So I suggest we add something like this to the doc: "Note that `hostPath` files or directories are only writable by root. You should make sure Alluxio worker has access to the tiered storage."